CMakeLists.txt

```diff
@@ -141,6 +147,15 @@ endif ()
 
 install (TARGETS iwasm_static ARCHIVE DESTINATION lib)
 
+# If it's a Release build, strip the static library
+if (CMAKE_BUILD_TYPE STREQUAL "Release")
+  # Strip static library
+  message(STATUS "Stripping static library after build!")
+  add_custom_command(TARGET iwasm_static POST_BUILD
+      COMMAND ${CMAKE_STRIP} ${CMAKE_C_STRIP_FLAGS} $<TARGET_FILE:iwasm_static>
```

Comment:
Here `${CMAKE_C_STRIP_FLAGS}` is also set to "${CMAKE_STRIP}", see L112, so the command is `${CMAKE_STRIP} ${CMAKE_STRIP} $<TARGET_FILE:iwasm_static>`, seems strange? Suppose that ${CMAKE_STRIP} is `strip`

Reply:
You are right sorry, then I set `CMAKE_STRIP_FLAGS ` as `"--strip-all"` and remove language dependency naming convention. Could you please review it again?